packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -3382,6 +3384,1125 @@ void main() {
       expect(find.byType(CupertinoButton), isContextMenuProvidedByPlatform ? findsNothing : findsNWidgets(3));
   }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS }));
 
+  group('Triple tap/click', () {
+    testWidgets(
+      'Can triple tap to select a paragraph on mobile platforms when tapping at a word edge',
+      (WidgetTester tester) async {
+        // TODO(Renzo-Olivares): Enable, currently broken because selection overlay blocks the tapanddraggesturerecognizer.
```

Comment:
Is your plan to do this in another PR?

Reply:
I'm hoping that I can fix it in this one. But if the fix becomes overly complex then I'll probably make it into another PR.

Reply:
Just wanted to draw attention to this again in case you forgot about it.